zigConsole/Dockerfile

```diff
@@ -17,45 +17,21 @@ ARG APP_ROOT=
 
 
 # BUILD ------------------------------------------------------------------------
-FROM torizon/cross-toolchain-${IMAGE_ARCH}:${SDK_BASE_VERSION} AS build
+FROM kassany/bookworm-ziglang:${SDK_BASE_VERSION} AS build
```

Comment:
Hello @kassane,
When running the `Dockerfile` in the way it was, I was running into the following issue during build:
```
...
 => [zigTesr build 7/9] RUN rm -rf /home/torizon/app/target                                                                                                                                                  0.1s 
 => ERROR [zigtest build 8/9] RUN if [ "arm64" = "arm64" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=aarch64-linux-gnu --summary all -freference-trace ;     elif [ "arm64" = "armhf" ] ;   49.9s 
------                                                                                                                                                                                                           
 > [zigtest build 8/9] RUN if [ "arm64" = "arm64" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=aarch64-linux-gnu --summary all -freference-trace ;     elif [ "arm64" = "armhf" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=arm-linux-gnu --summary all -freference-trace ;     elif [ "arm64" = "amd64" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=x86_64-linux-gnu --summary all -freference-trace ;     fi:                                                                                                                                                                                      
49.71 Illegal instruction at address 0x1156abc                                                                                                                                                                   
49.71 Panicked during a panic. Aborting.                                                                                                                                                                         
49.72 qemu: uncaught target signal 6 (Aborted) - core dumped
49.82 error: the following build command crashed:
49.82 /home/torizon/app/.zig-cache/o/94cec820d9640d024730db2cf8a14d9f/build /zig/0.13.0/files/zig /home/torizon/app /home/torizon/app/.zig-cache /root/.cache/zig --seed 0xc9a9e621 -Z3a3bc4625ad946be -Doptimize[+] Building 0/1target=aarch64-linux-gnu --summary all -freference-trace
 ⠙ Service zigtest  Building                                                                                                                                                                                52.1s 
failed to solve: process "/bin/sh -c if [ \"$IMAGE_ARCH\" = \"arm64\" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=aarch64-linux-gnu --summary all -freference-trace ;     elif [ \"$IMAGE_ARCH\" = \"armhf\" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=arm-linux-gnu --summary all -freference-trace ;     elif [ \"$IMAGE_ARCH\" = \"amd64\" ] ; then         zig build -Doptimize=ReleaseSmall -Dtarget=x86_64-linux-gnu --summary all -freference-trace ;     fi" did not complete successfully: exit code: 1

 *  The terminal process "/usr/bin/bash '-c', 'DOCKER_HOST= docker compose build --pull --build-arg SSHUSERNAME=torizon --build-arg APP_ROOT=/home/torizon/app --build-arg IMAGE_ARCH=arm64 --build-arg GPU=-imx8 zigtest'" terminated with exit code: 17. 
 ```
 So I updated the base container of the build step to use the same one as in the `Dockerfile.sdk` and it worked